oshi-core/src/main/java/oshi/hardware/HWPartition.java

```diff
@@ -350,4 +351,17 @@ public boolean equals(Object obj) {
         return true;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(" |-- ").append(getIdentification()).append(": ");
+        sb.append(getName()).append(" ");
+        sb.append("(").append(getType()).append(") ");
+        sb.append("Maj:Min=").append(getMajor()).append(":").append(getMinor()).append(", ");
+        sb.append("size: ").append(FormatUtil.formatBytesDecimal(getSize()));
+        sb.append(getMountPoint().isEmpty() ? "" : " @ " + getMountPoint()).append("\n");
```

Comment:
And similar to earlier, don't include the newline in the toString. If needed for the output, add in the other class (but it may not be needed there).